lib/cli/extension-command.js

```diff
@@ -486,4 +487,49 @@ export default class ExtensionCommand {
     this.config.installedExtensions[ext].version = version;
     await this.config.write();
   }
-}
+
+  /**
+   * Runs a script cached inside the "scripts" field under "appium"
+   * inside of the driver/plugins "package.json" file
+   *
+   * @param {string} ext - name of the extension to run a script from
+   * @param {string} scriptName - name of the script to run
+   */
+  async run ({ext, scriptName}) {
+    if (!_.has(this.config.installedExtensions, ext)) {
+      const msg = `please install the ${this.type} first`;
+      throw new Error(msg);
+    }
+
+    const extConfig = this.config.installedExtensions[ext];
+
+    if (!_.has(extConfig, 'scripts')) {
+      throw new Error(`The ${this.type} named '${ext}' does not contain the ` +
+                      `"scripts" field underneath the "appium" field in its package.json`);
+    }
+
+    const extScripts = extConfig.scripts;
+
+    if (!(scriptName in extScripts)) {
+      throw new Error(`The ${this.type} named '${ext}' does not support the script: '${scriptName}'`);
+    }
+
+    const runner = new SubProcess(`node`, [extScripts[scriptName]],
+                                  {cwd: this.config.getExtensionRequirePath(ext)});
+
+    runner.on('stream-line', (line) => {
+      log(this.isJsonOutput, line);
+    });
+
+    await runner.start();
+
+    try {
+      await runner.join();
+      log(this.isJsonOutput, `${scriptName} successfuly ran`.green);
+      return {scriptName};
+    } catch (err) {
+      log(this.isJsonOutput, `Encountered an error when running the script: ${err.message}`.red);
+      return {error: err.message};
```

Comment:
in addition to the error message, it would be nice to include any output in the JSON response, since the error message might be something generic like 'exited with code 1'.